### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hsachdevah**
Modified Readme file

### CODE REVIEW

File: applications/jupyter/README.md [hunk 1]
```diff
@@ -2,125 +2,91 @@
...
 
-Continue to Step 3 of [below](#if-auth-is-enabled).
+> **_NOTE:_** If there isn't an external IP for `proxy-public`, is it most likely due to authencation being enabled.
 
 ### If Auth is enabled
 
-1. Run `kubectl describe managedcertificate managed-cert -n <namespace>`. The namespace is the user name that you specified above.
-
-2. Copy the domain under `Domains`
+1. Copy the value for `domain` from terraform output section.
```
Comment (imreddy13) [inline applications/jupyter/README.md:85]: what is terraform output section? is that what is printed after running terraform apply based on outputs.tf? What if they come back to this at a later date, how do they find the domain? Is it visible anywhere in pantheon?

File: applications/jupyter/README.md [hunk 2]
```diff
@@ -171,6 +137,6 @@ We allow user to set their own domains, in the `variables.tf` file. Since we are
```
Comment (imreddy13) [inline applications/jupyter/README.md:137]: Should this be workloads.tfvars?

Comment (imreddy13) [inline applications/jupyter/README.md:137]: What does this mean (the ingress thing)?

File: applications/jupyter/README.md [hunk 3]
```diff
@@ -2,125 +2,91 @@
...
-7. Run `gcloud container clusters get-credentials %gke_cluster_name% --location=%location%`
-   Configuring `gcloud` [instructions](https://cloud.google.com/sdk/docs/initializing)
+**Important Note:**
+If using this with the Ray module (`applications/ray/`), it is recommended to use the same k8s namespace
+for both i.e. set this to the same namespace as `applications/ray/variables.tf`.
 
-8. Run `terraform apply`
+3. If you have not enabled the IAP API before or created a Brand for your project, please follow these steps:
 
-## Securing your Jupyter Endpoint
```
[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md:77]: Created sub sections within "Installation"

File: applications/jupyter/README.md [hunk 4]
```diff
@@ -2,125 +2,91 @@
...
+    - Navigate to the `brand` [page](https://console.cloud.google.com/apis/credentials/consent) to create your own brand:
 
-To secure the Jupyter endpoint, this example enables IAP by default. It is _strongly recommended_ to keep this configuration. If you wish to disable it, do the following: set the `add_auth` flag to false in the `variable.tf` file.
+    Visit [here](#auto-brand-creation-and-iap-enablement) for more information about how to create a brand automatically.
```
[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md:55]: Added details & links

File: applications/jupyter/README.md [hunk 5]
```diff
@@ -2,125 +2,91 @@
...
 
-    **Note:** Seperate each principals with a new line
+### JupyterHub
 
-6. Find the name and location of the GKE cluster you want to use.
-   Run `gcloud container clusters list --project=<your GCP project> to see all the available clusters.
+1. If needed, git clone https://github.com/GoogleCloudPlatform/ai-on-gke
 
-   Note: If you created the GKE cluster via the ai-on-gke/gke-platform repo, you can get the cluster info from `ai-on-gke/gke-platform/variables.tf`
+2. Edit `variables.tf` with your GCP settings. The `namespace` that you specify will become a K8s namespace for your Jupyterhub services. For more information about what the variabls do visit [here](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/applications/jupyter/variable_definitions.md)
```
[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md:45]: Fixed this & few more typos

[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md:45]: This was wrongly written at multiple places. Fixed all of them.

File: applications/jupyter/README.md (hunk 1 above)
Comment (hsachdevah) [reply applications/jupyter/README.md]: Added reference to previous step & also to the managed certificate page where domain value can be obtained from

File: applications/jupyter/README.md [hunk 6]
```diff
@@ -2,125 +2,91 @@
...
+6. Run `terraform apply --var-file=./workloads.tfvars`
 
-11. Re-run `terraform apply`
+7. Note down the value for `domain` from terraform output section. You can open this in a browser & login with your credentials 
 
-12. Navigate to the [GCP IAP Cloud Console](https://console.cloud.google.com/security/iap) and select your backend-service checkbox.
+> **_NOTE:_** Domain specefic managed certificate may take some time to finish provisioning. On average around 10-15 minutes. 
 
-13. Click on `Add Principal`, insert the new principle and select under `Cloud IAP` with role `IAP-secured Web App User`
+8. Navigate to the [GCP IAP Cloud Console](https://console.cloud.google.com/security/iap) and select your backend-service checkbox.
```
[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md:91]: added backend service name `<namespace>/proxy-public`

File: applications/jupyter/README.md (hunk 2 above)
Comment (hsachdevah) [reply applications/jupyter/README.md:186]: Updated statement

File: applications/jupyter/README.md [hunk 7]
```diff
@@ -2,125 +2,91 @@
...
 
-> **_NOTE:_** Your managed certificate may take some time to finish provisioning. On average around 10-15 minutes. 
+9. Click on `Add Principal`, insert the new principle and select under `Cloud IAP` with role `IAP-secured Web App User`
```
[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md]: Updated instructions

File: applications/jupyter/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md]: Restructured the instructions to explain the sequence better.

[the remark this answers is not in the document]
Comment (hsachdevah) [reply applications/jupyter/README.md]: Added example workload files

**Review by imreddy13 — APPROVED** [review]

**Pull request merged**